examples/transaction.rs

```diff
@@ -100,3 +106,64 @@ fn main() {
     let key2: Key = b"key2".to_vec().into();
     dels(&txn, vec![key1, key2]);
 }
+
+fn parse_args() -> (Vec<String>, Option<(PathBuf, PathBuf, PathBuf)>) {
```

Comment:
duplicated code here, can we abstract them to a common mod so all examples can use it?

Reply:
Most duplicated code is extracted to a common mod. PTAL.